omero/sysadmins/server-backup-and-restore.txt

```diff
@@ -79,7 +78,7 @@ OMERO.server has three main backup sources:
 You should back up *(1)* and *(2)* regularly. 
 
 .. warning:: **Although losing your PostgreSQL database is not as 
```

Comment:
I do wonder if we need this paragraph at all.  There's nothing particularly informative or actionable here.  And what happened in the 4.4 timeframe is franky not relevant or useful information.  And if you upgraded from a 4.4 or earlier system, a database loss is as catastrophic for data imported back then as it was then, so the advice is bad anyway.